### ISSUE TITLE
Eigen includes should not contain version prefix

### ISSUE THREAD

**Issue by gavanderhoorn** [shaun-edwards/moveit_simple#33]
The current `#include`s seem to have the `eigen3` part included:

https://github.com/shaun-edwards/moveit_simple/blob/2bcca7d855c199883963c81b4f6668b060533401/moveit_simple/include/moveit_simple/moveit_simple.h#L28

That is rather unusual, and will limit future compatibility with (if it ever gets released) v4 of Eigen.

Was this done to work around incorrectly setup include paths (ie: only `/usr/include` on the path)?

**Comment by MauleshSTrivedi** [comment]
@gavanderhoorn Good catch. It was definitely a part of an old commit, but I'm not sure why this was done specifically. I can make the necessary changes once I have more information regarding this.
